src/msg.h

```diff
@@ -87,6 +87,8 @@
 #define MSG_CFG_LOAD_FAIL       MSG_PREFIX "0072E" " Launcher Could not load configurations\n"
 #define MSG_CFG_SCHEMA_FAIL     MSG_PREFIX "0073E" " Launcher Could not load schemas, status=%d\n"
 #define MSG_NO_LOG_CONTEXT      MSG_PREFIX "0074E" " Log context was not created\n"
+#define MSG_IPC_CLEANUP_FAILED  MSG_PREFIX "0075D" " Automatic IPC Cleanup Failed.\n"
+#define MSG_IPC_CLEANUP_SUCCESS MSG_PREFIX "0076D" " Automatic IPC Cleanup Succeeded.\n"
```

Comment:
1. I would be consistent with the rest of the IDs, Error for 0075E and Info for 0076I.
2. The doc change should be included in this file, corresponding update in markdown starting as comment `//@`.

Reply:
I'll keep the success message as an INFO message, and embed the failure message as a debug statement. If the command fails, it's not fatal and the user doesn't necessarily need to take action, so I don't want to add another 'failure' message which is purely informational and could cause confusion. When we work with users, we can check for the success message in the INFO logs or ask them to enable debug.